### PR TITLE
[installation] fix installation and some name typo

### PR DESCRIPTION
**Pull request by TheaperDeng**
### 1. Motivation and Context

1. Change the installation option from `[all]` to `[fast_jl]` since it's the only optional package among the dependencies.
2. Rename an example filename to better reflect the content

### 2. Summary of the change

See motivation

### 3. What tests have been added/updated for the change?
- [ ] Unit test: Typically, this should be included if you implemented a new function/fixed a bug.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,20 +26,20 @@ git clone https://github.com/TRAIS-Lab/dattri
 pip install -e .
 ```
 
-If you want to use all features on CUDA and accelerate the library, you may install the full version by
+If you want to use `fast_jl` to accelerate the random projection, you may install the full version by
```
Comment (jiaqima) [inline README.md:29]: "the full version" -> "the version with `fast_jl`"

File: dattri/metric/ground_truth.py [hunk 2]
```diff
@@ -160,7 +160,8 @@ def target_func(ckpt_path, dataloader):
             target function calculated on all test samples under `num_subsets` models,
             each retrained on a subset of the training data. The second tensor has the
             shape (num_subsets, subset_size), where each row refers to the indices of
-            the training samples used to retrain the model.
+            the training samples used to retrain the model. The targeted value will be
```
Comment (jiaqima) [inline dattri/metric/ground_truth.py:163]: targeted -> target

**Review by jiaqima — APPROVED** [review]
Otherwise LGTM
